requirements.txt

```diff
@@ -12,7 +15,7 @@ spacy>=3.1.4,<=3.3.0
 datasketch~=1.5.6
 
 # utility
-wasabi~=0.9.0
+wasabi~=0.8.2
```

Comment:
Is there a problem with version 0.9.0?

Reply:
Conflict with dacy 1.3.0 - ERROR: dacy 1.3.0 has requirement wasabi<0.9.0,>=0.8.2, but you'll have wasabi 0.9.0 which is incompatible.

Reply:
Ahh. Jamen hold den på 0.8.2. Der er en PR på vej til DaCy - den er dog ikke som priority lige nu.